fuzzers/baby_fuzzer_grimoire/src/main.rs

```diff
@@ -15,7 +15,7 @@ use libafl::{
         GrimoireRandomDeleteMutator, GrimoireRecursiveReplacementMutator,
         GrimoireStringReplacementMutator, Tokens,
     },
-    observers::StdMapObserver,
+    observers::{StdMapObserver, TrackingHinted},
```

Comment:
I think `Hinted` isn't a great name, it's the _how_ not the _what_, right? Maybe `EnableTracking` or something?

Reply:
Or `CanTrack` lol

Reply:
`HintsTracking` also feels gross.

Reply:
I like `CanTrack`, sounds like `CanBus` (and is in line with `HasXYZ`)

Reply:
If it `CanTrack` you can call `track_whatever` on the guy

Reply:
CanTrack seems the best out of these options. I'll do a mass rename once CI is clear.

Reply:
Maybe even a `CanTrackX` for each individual trackable feature?

Reply:
Hm, I don't think this is desirable. Splitting this trait further doesn't help us, since map observers will always implement both.

Reply:
Ok, I was just thinking we might have a `CanTrack`for  something else later

Reply:
Rename?